changes/tasks.py

```diff
@@ -524,6 +560,17 @@ def check_momconnect_change_language(self, data_fields, change):
         else:
             return []
 
+    def check_momconnect_change_msisdn(self, data_fields, change):
+        """
+        Ensures that a new msisdn is provided, and that it is a valid msisdn.
+        """
+        if "msisdn" not in data_fields:
```

Comment:
The tricky bit here is that it's difficult to know what an identity's old address is, and it could also be multiple addresses if there's no or more than one default set. Should we just be storing the whole dictionary of addresses? Or should we specifically look at which address/addresses and just store those?

Technically, since it doesn't remove any addresses, the old address will still be there. What we can't tell is which ones, if any, were default addresses before we made the change.